configuration/openmrs/apps/customDisplayControl/js/customControl.js

```diff
@@ -545,7 +545,7 @@ function billingStatusController($scope, $element, erpService, visitService, app
         } else if (orderLine.qty_invoiced > 0 && orderLine.qty_to_invoice > 0) {
           // PARTIALLY_INVOICED
           tags.push(PARTIALLY_INVOICED);
-        } else if (orderLine.qty_to_invoice <= 0) {
+        } else if (orderLine.qty_to_invoice == 0) {
```

Comment:
Should you remove the comment line below if we're not checking for negative quantities?